data/types/food/nutrition.go

```diff
@@ -32,15 +32,15 @@ func NewNutrition() *Nutrition {
 }
 
 func (n *Nutrition) Parse(parser structure.ObjectParser) {
-	n.AbsorptionDuration = parser.Int("absorptionDuration")
+	n.EstimatedAbsorptionDuration = parser.Int("absorptionDuration")
```

Comment:
Do you need to change the string also?

Reply:
Congrats - you definitely found a real issued!  Thanks